### PR TITLE
Changed the message for connection issues

### LINKED ISSUE #352: Allow users to contact support when they have internet and cannot connect to Teton servers
Large companies typically block access to external servers. We have seen a few users who cannot login after installation because the connection to our server is blocked by their IT.

We should issue a message to the user when this happens, and tell them our servers are being blocked, and that they should contact us.

### PR DESCRIPTION
**Pull request by jake-kirkland-teton**
Added more text and a link to the help page.
Additionally, fixed a bug with the new messages system.

Had a little trouble with this one getting the hover hand working correctly, and discovered a bug with the messaging system that I'm honestly not sure how it was working at all without. Should be good to go now.

Closes #352

### CODE REVIEW

**Review by dougkenik — CHANGES_REQUESTED** [review]
A few comments in the picture below on my machine:

- "Servers" should not be capitalized. Add an Exclamation point at the end
- I am seeing text wrapped in <href>. We need to get rid of that, and.... we need to have the link open the user's email client addressed to help@tetonsim.com, with a prepopulated title "Smart Slice Cannot Connect Error" A reference of this is in Extensions->Smart Slice->Contact (I think I gave you the wrong link last time)



Untitled

File: SmartSlicePlugin/SmartSliceCloudConnector.py [hunk 1]
```diff
@@ -559,7 +560,7 @@ def _handleThorErrors(self, http_error_code, returned_object):
         if self._error_message is not None:
             self._error_message.hide()
 
-        self._error_message = Message(lifetime= 180)
+        self._error_message = SmartSliceMessage(smartslice_lifetime= 180)
```
Comment (bradymadams) [inline SmartSlicePlugin/SmartSliceCloudConnector.py:563]: Styling - no spaces around `=` on named argument assignments.

**Review by bradymadams — CHANGES_REQUESTED** [review]
Doug already mentioned the major items. I just have one style change request.

File: SmartSlicePlugin/SmartSliceCloudConnector.py (hunk 1 above)
Comment (jake-kirkland-teton) [reply SmartSlicePlugin/SmartSliceCloudConnector.py:563]: Got it, good catch.

**Comment by jake-kirkland-teton** [comment]
Alright, check again for me. I'm not sure why the inline worked for me and not you Doug. 

About the text wrapping, does it look ok now that the link is hopefully fixed?

And ahhh. No worries. I got it changed to a mailto.

**Review by dougkenik — CHANGES_REQUESTED** [review]
Still not displaying correctly. Let's also remove the comma between "connection, and are still"

image

**Comment by jake-kirkland-teton** [comment]
OK. Please check again and make sure it is appearing correctly for you guys, but I think that should do it.

File: SmartSlicePlugin/SmartSliceCloudConnector.py (hunk 1 above)
Comment (bradymadams) [reply SmartSlicePlugin/SmartSliceCloudConnector.py:563]: Looks like this still needs to be fixed.

**Review by bradymadams — APPROVED** [review]

**Comment by jake-kirkland-teton** [comment]
Not sure how that one got lost, but my fault. Got it for sure now.

**Review by dougkenik — APPROVED** [review]

**Pull request merged**